packages/modules/create-imodel/src/components/update-imodel/UpdateIModel.test.tsx

```diff
@@ -82,6 +83,76 @@ describe("UpdateIModel", () => {
     );
   });
 
+  it("should enable update when extent is removed", async () => {
+    const successMock = jest.fn();
+    toaster.positive = jest.fn();
+
+    const { getByText, rerender } = render(
+      <UpdateIModel
+        accessToken="dd"
+        imodelId="de47c5ad-5657-42b8-a2bc-f2b8bf84cd4b"
+        onSuccess={successMock}
+        apiOverrides={{ serverEnvironmentPrefix: "dev" }}
+        initialIModel={{
+          name: "Initial name",
+          description: "Initial description",
+          extent: {
+            southWest: { latitude: 1, longitude: 2 },
+            northEast: { latitude: 3, longitude: 4 },
+          },
+        }}
+      />
+    );
+
+    const updateButton = getByText("Update");
+    expect(updateButton.closest("button")?.hasAttribute("disabled")).toBe(true);
```

Comment:
Why `closest` is needed?
I think you can write:
```tsx
    const updateButton = getByText("Update") as HTMLButtonElement;
    expect(updateButton.disabled).toBe(true);
```

Reply:
The update text is present on the span element, so we require closest to first access the button element and then find the disabled attribute on it.

Reply:
Oh wow, I didn't know, that is so bad that you can't just query a button by text...